app/controllers/orders_controller.rb

```diff
@@ -34,12 +33,32 @@ def order_params
 
   def create_item_orders(order)
     cart.items.each do |item, quantity|
-      order.item_orders.create({
+      if check_item_for_discounts(item, quantity)
+        create_item_order(order, item, quantity)
+      else 
+        create_discounted_item_order(order, item, quantity)
+      end
+    end
+  end
+
+  def create_item_order(order, item, quantity)
+    order.item_orders.create({
         item: item,
         quantity: quantity,
         price: item.price
         })
-    end
+  end
+
+  def create_discounted_item_order(order, item, quantity)
```

Comment:
i could potentially combine this into one method because it is almost the same method as the one above, so might look into that when refactoring